ksqldb-engine/src/test/java/io/confluent/ksql/planner/plan/AggregateNodeTest.java

```diff
@@ -306,7 +306,8 @@ public void shouldCreateLoggers() {
     assertThat(loggers, contains(
         "queryid.KsqlTopic.Source",
         "queryid.Aggregate.GroupBy",
-        "queryid.Aggregate.Aggregate.Materialize"
+        "queryid.Aggregate.Aggregate.Materialize",
+        "queryid.Aggregate.Project"
```

Comment:
I don't totally understand why this changed yet (I assume it has to do with the changes in TableSelectBuilder?) but at the surface this looks concerning. I wouldn't have expected this PR to affect aggregates at all.

Reply:
In this test, we just parse an aggregation query that inserts a `TableSelect` upstream that we need to mock.